docs/application/dotnet/guides/uix/tts.md

```diff
@@ -478,18 +478,18 @@ To set and get TTS engine options:
     ```
 
 <a name="text"></a>
-## Adding Text
+## Add Text
```

Comment:
I think the second heading capitalization issue was not addressed.
```suggestion
## Add text
```

Reply:
Thank you for checking.
I also missed it.
I will fix it.